docs/user/connected-accounts.rst

```diff
@@ -1,28 +1,51 @@
-Connecting Your VCS Account
-===========================
+Connecting your Git repository
+==============================
+
+In this article, we explain how connecting your Read the Docs account to one of the supported Git providers
+automatically configures your Git repository and your Read the Docs project.
+
+✅️ Signed up with your Git provider?
+  If you signed up or logged in to Read the Docs with your GitHub, Bitbucket, or GitLab
+  credentials, you're all done. Your account is connected and the integration explained
+  below is enabled.
+⬇️ Signed up with your email address?
+  If you have signed up to Read the Docs with your email address,
+  you can add the connection to the Git provider afterwards.
+  Read more about how it works below.
+
 
 If you are going to import repositories from GitHub, Bitbucket, or GitLab,
-you should connect your Read the Docs account to your repository host first.
+we recommend that you connect your Read the Docs account to your repository provider.
```

Comment:
We are using different names to refer to the same thing in these documents.

* repository provider
* git provider
* repository host

I'd say we should use only one to refer to this because it may confuse the users.

Reply:
Thanks! Yes, this decision should really be made now that we have a Style Guide to consolidate it in: https://dev.readthedocs.io/en/latest/style-guide.html#word-list

We decided to say "Git repository" instead of VCS, but we didn't add anything that reasonably covers the "platform/host/provider" in the Style Guide.

I'll forward this in the open PR for the style guide.. adding a task for this PR to implement whatever we decide :+1:

Reply:
Added a suggestion for using "Git provider" to https://github.com/readthedocs/readthedocs.org/pull/9840 - let's decide there.